examples/pulse_control/1_Bringup_experiments.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>The data is then**</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>